docs/configuration.rst

```diff
@@ -69,6 +69,10 @@ The ``RELAYHOST`` is an optional address of a mail server relaying all outgoing
 mail in following format: ``[HOST]:PORT``.
 ``RELAYUSER`` and ``RELAYPASSWORD`` can be used when authentication is needed.
 
+By default postfix uses "opportunistic TLS" for outbound mail. This can be changed
+by setting ``OUTBOUND_TLS_LEVEL`` to ``encrypt``. This setting is highly recommended
+if you are a relayhost that supports TLS.
```

Comment:
1. Are `may` and `encrypt` the only options available? If there are other options, they need to be documented. Perhaps also make clear that the default value is set to `may`.

> This setting is highly recommended if you are a relayhost that supports TLS.

2. This wording is a bit strange. I would say "if you have a relayhost" or "if this is a relayhost".

Reply:
Right, should have been "if you are using a ralayhost" ;)

Reply:
>     1. Are `may` and `encrypt` the only options available? If there are other options, they need to be documented. Perhaps also make clear that the default value is set to `may`.
> 
Another options would be DANE, when #707  is fixed

Reply:
Okay, we should take care of documenting that issue when the time is there.

Reply:
Additional options are documented on http://www.postfix.org/postconf.5.html#smtp_tls_security_level  … however, i think that `may` or `encrypt` (and later dane) make most sense for a typical simplistic mailu deployment. Thus, i would not bloat documentation here by listing all of them. For me, it’s fine like this.